Bottom: 594eefe67fc7781d81ea5ce8bf1fe7d5f0027013
Top:    9e98fc86d59414946590790810e290822966624a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:33:31 +0100

Refresh of draw-c-split-gathering-from-re

---

diff --git a/src/draw.c b/src/draw.c
index eb79bed..5c34b25 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -54,6 +54,9 @@
 #include <dmalloc.h>
 #endif
 
+#undef NDEBUG
+#include <assert.h>
+
 #ifndef MAXINT
 #define MAXINT (((unsigned int)(~0))>>1)
 #endif
@@ -1147,6 +1150,8 @@ DrawArcLowLevel (ArcTypePtr Arc)
 static void
 DrawElementPackageLowLevel (ElementTypePtr Element)
 {
+  assert (Gathering);
+
   /* draw lines, arcs, text and pins */
   ELEMENTLINE_LOOP (Element);
   {
